packages/library-base/src/commonMain/kotlin/io/realm/Configuration.kt

```diff
@@ -258,6 +290,26 @@ interface Configuration {
         fun encryptionKey(encryptionKey: ByteArray) =
             apply { this.encryptionKey = validateEncryptionKey(encryptionKey) } as S
 
+        /**
+         * Due to the way Realm allocates space on disk, it is sometimes the case that more space
```

Comment:
A bit funny first sentence. Maybe something like "Set callback for controlling whether the realm should be compacted on open. Due to the ... "